UnitTests/JsonStat2/JsonStat2SerializerTests.cs

```diff
@@ -0,0 +1,68 @@
+using System.Globalization;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Newtonsoft.Json.Linq;
+
+using PCAxis.Paxiom;
+
+
+namespace UnitTests.JsonStat2
+{
+    [TestClass]
+    [DeploymentItem("TestFiles/HalfYearStatistics.px")]
+    public class JsonStat2SerializerTests
+    {
+        private JObject _jsonstat;
+
+        [TestInitialize]
+        public void TestSetup()
+        {
+            var helper = new JsonStat2Helper();
+
+            // CultureInfo ci = new("fi-FI");
+            // CultureInfo ci = new("no-NO");
+            // CultureInfo ci = new("sv-SE");
+            // System.Threading.Thread.CurrentThread.CurrentCulture = ci;
+            // System.Threading.Thread.CurrentThread.CurrentUICulture = ci;
+
+            PXModel myModel = helper.GetSelectAllModel("HalfYearStatistics.px");
+
+            var actual = helper.GetActual(myModel);
+
+            _jsonstat = JObject.Parse(actual);
+        }
+
+        [TestMethod]
+        public void Check_RootLevel_Elements()
```

Comment:
I also tried to add tests to the exciting code and I struggle with `label` and `updated`